GrandCentralBoard/Widgets/Bonus/Source/PeopleWithBonususFetchController.swift

```diff
@@ -0,0 +1,134 @@
+//
+//  BonusFetchController.swift
+//  GrandCentralBoard
+//
+//  Created by Rafal Augustyniak on 07/04/16.
+//  Copyright © 2016 Oktawian Chojnacki. All rights reserved.
+//
+
+import Foundation
+import GrandCentralBoardCore
+
+
+private let kPreferredPeopleCount = 10
+
+enum PeopleWithBonususFetchControllerError : ErrorType {
+    case IncorrectEmailAddress
+    case Cancelled
+    case Unknown
+}
+
+
+final class PeopleWithBonususFetchController {
+
+    private let requestSender: RequestSender
+
+    init(requestSender: RequestSender) {
+        self.requestSender = requestSender
+    }
+
+    func fetchPeopleWithBonuses(completionBlock: (Result<[Person]>) -> Void) {
+        fetchPeopleWithBonuses(startingFromDate: NSDate.init(), fetchedBonuses: [], completionBlock: { [weak self] result in
+            guard let strongSelf = self else { return }
```

Comment:
CompletionBlock not called ( but should it be, if the object is deallocated? I would say it's good to always call completion block)